index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="Drag Race Data" />
+    <meta property="og:description" content="API with all the Drag Race data" />
+
+    <meta
+      property="og:image"
+      content="https://dragrace-data.herokuapp.com/kandymuse.png/"
+    />
+    <link rel="shortcut icon" href="kandy.png" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+    <title>drag race data</title>
+  </head>
+  <body>
+    <div
+      class="gif"
+      style="width: 100%; height: 0; padding-bottom: 56%; position: relative"
+    >
+      <iframe
```

Comment:
I'm impressed that you sent an HTML file into the API to display as the front page!